### ISSUE TITLE
Insufficient list generated on Debian Jessie

### ISSUE THREAD

**Issue by embash** [mglinux/afwall_easy#1]
Hi, thanks for your work on this. The aafwall.sh file generated on my system does not contain the necessary $IPTABLES -A "afwall" entries.

Please note:
- System is Debian Jessie, upgraded to latest point release.
- Packages python (2.7.9-1) and python-ipaddress (1.0.16-1~bpo8+1) are installed.
- The afwall.sh script is executable.

Running the script with the following selections:
```
[3] 91.239.100.100 Censurfridns Denmark
Block Google? y
Block Facebook? y
Block Samsung? y
Block Apple? y
Block Amazon? n
Block Microsoft? y
```
generates the following entries at the end of afwscripts/aafwall.sh (see attachment [aafwall.sh.txt](https://github.com/mglinux/afwall_easy/files/406442/aafwall.sh.txt) ):
```
# DNS Server and custom rules generated from the script
$IPTABLES -t nat -I OUTPUT -p tcp --dport 53 -j DNAT --to-destination 91.239.100.100:53
$IPTABLES -t nat -I OUTPUT -p udp --dport 53 -j DNAT --to-destination 91.239.100.100:53
$IPTABLES -A afwall --destination 104.197.59.8 -j RETURN
```
There should be a lot more entries. I would like to fix this of course, but also to suggest that you modify the script documentation with example output so that if this happens to naive users, they don't merrily run the results and wonder why nothing gets blocked.

Kind regards,
Mike

**Comment by embash** [comment]
Okay, I found the source of my issue. Using my incredibly limited Python skills I figured out that the ipaddr module was not available on my system, despite having the python-ipaddress package installed. For example, executing `from ipaddr import IPv4Network, CollapseAddrList` in python2 resulted in a module error. Changing it to ipaddress caused it to complain that CollapseAddrList is not available. (I think it was loading the python3 module.)

After more searching, I found the Debian package python-ipaddr. After installing it, the module is available and your script creates files like facebook0.

I therefore suggest modifying the documentation to warn users to make sure they have the right module installed, in addition to sample output.

Thanks again,
Mike
